In some setups firefox crashes because of wrongly aligned variables
Observed on a ARM Cortex-A9 target.

Upstream-Status: Pending

Signed-off-by: Max Krummenacher <max.oss.09 at gmail.com>
--- mozilla-release/ipc/ipdl/ipdl/cxx/cgen.py~	2012-07-29 11:14:45.000000000 +0200
+++ mozilla-release/ipc/ipdl/ipdl/cxx/cgen.py	2012-07-29 11:13:36.000000000 +0200
@@ -107,7 +107,7 @@
             self.println(';')
         self.dedent()
 
-        self.printdent('}')
+        self.printdent('} __attribute__ ((aligned (__BIGGEST_ALIGNMENT__)))')
 
 
     def visitTypedef(self, td):
